index.js

```diff
@@ -134,44 +143,70 @@ const listInMainPart = {
                 {view: "button", id: "btn-sort-desc", value: "Sort desc", css: "webix_primary", width: 150, click:function(){
                         $$("list-users").sort("#name#","desc");
                     }},
+                {view: "button", id: "btn-add-user", value: "Add User", css: "webix_primary", width: 150, click:function(){
+                        const age = Math.floor(Math.random() * 100) + 1;
+                        $$("list-users").add({name:"User", country:"USA", age: age});
```

Comment:
if there are many items, new ones are added to the end of data, so that users do not see them. try to:

- either add new items to the beginning
- or scroll list by API to show the new item